api/api_authorizations.gen.go

```diff
@@ -208,7 +208,7 @@ func (a *AuthorizationsApiService) DeleteAuthorizationsIDExecuteWithHttpInfo(r A
 		return nil, GenericOpenAPIError{error: err.Error()}
 	}
 
-	localVarPath := localBasePath + "/authorizations/{authID}"
+	localVarPath := localBasePath + "/api/v2/authorizations/{authID}"
```

Comment:
These changes are suprising - why?

Reply:
The problem was that the `/query` endpoint in `cli.yml` was used at the root but with the root server as `/api/v2`, but the legacy InfluxQL endpoint is just `/query`. I couldn't have two `/query` endpoints so I needed to move all the `/api/v2` endpoints so that the `/query` becomes `/api/v2/query` and the legacy `/query` can be distinct.
Maybe it wasn't necessary this way, I'm new to using openapi schemas.